src/assets/scss/pages/story.scss

```diff
@@ -1,11 +1,36 @@
 .author-information {
+  align-items: center;
   display: flex;
   flex-direction: row;
-  margin-left: 3em;
+  flex-wrap: wrap;
+  justify-content: flex-start;
   margin-top: 0.5em;
 
+  & > * {
+    margin-right: 4rem;
```

Comment:
let's reduce it to 3rem please, and let's also add `:not(:last-child)` so that the last child doesn't inherit the properties.

it should look like:

```
& > *:not(:last-child) {
    margin-right: 4rem;
  } 
 ```

Reply:
Done